tools/test.py

```diff
@@ -142,7 +142,7 @@ def Run(self, tasks):
       # Wait for the remaining threads
       for thread in threads:
         # Use a timeout so that signals (ctrl-c) will be processed.
-        thread.join(timeout=10000000)
```

Comment:
It looks weird to me that this harmless value (neither INT_MAX or any power of 2) would cause an overflow!